tools/update-authors.js

```diff
@@ -11,6 +11,7 @@ class CaseIndifferentMap {
   _map = new Map();
 
   get(key) { return this._map.get(key.toLowerCase()); }
+  has(key) { return this._map.has(key.toLowerCase()); }
   set(key, value) { return this._map.set(key.toLowerCase(), value); }
```

Comment:
Shouldn't. we be using [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) if we are passing names in addition to email addresses?

Reply:
What makes you say that?